packages/core/src/config/config.ts

```diff
@@ -4,6 +4,8 @@ import trimEnd from 'lodash/trimEnd';
 import extend from 'lodash/extend';
 import isObject from 'lodash/isObject';
 import type { RequestContext } from '../api';
+import validUrl from 'valid-url';
```

Comment:
this is a total nit, so no worries either way. If you don't want to add a dep you can just use the native `URL` type to validate the URI. 

```js
try {
  const uri = new URL(suspect);
} catch (e) {
  // if the constructor errors int's an invalid url
}
```

Reply:
This is what I am thinking. valid-url is a good library, but it seems like overkill for what we need to do.

We also have [parseUrl](https://github.com/wpengine/headless-framework/blob/canary/packages/core/src/utils/convert.ts#L71). We could created an assertion function in `utils/assert.ts` called `isUrl` ti indicate whether something is a URL or just a string/path.

Reply:
Yeah good thinking here, I'll remove this dep